### ISSUE TITLE
Add export/import saved queries

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [prisma/prisma-admin-feedback#71]
Add export/import saved queries (e.g. to/from file).

**Comment by R4meau** [comment]
Yes! I'd like to be able to make multiples entries from a CSV file:
- The interface allows me to download a CSV template file that contains all the columns of the current query result. Or one that I've saved.
- I fill in the necessary fields
- I simply import that file to upload multiple entries at the same time.
- I'd like to still be able to use the form when it comes to a single entry.

As for the exporting, simply exporting the current query result as a CSV file sounds perfect to me.

**Comment by FluorescentHallucinogen** [comment]
@R4meau Originally, I meant export/import the queries themselves, not the result data.

**Comment by R4meau** [comment]
@FluorescentHallucinogen I see. Well, hopefully I'm not the only one who wants a data import and export feature. I'd pay for this.
